STYLEGUIDE.md

```diff
@@ -319,26 +303,6 @@ import inSibling from '../foo/child';
 Don't do this. Everything should be wrapped in a module that can be depended on
 by other modules. Even things as simple as a single value should be a module.
 
-### Function definitions
```

Comment:
I think the reasoning for this is no longer true. The dev tools and also React developer tools perfectly fine detect the name of the variable now as a function name in stack traces and debugging. So the reasoning is no longer true, and I think pretty much both are okay, it totally depends down to the situation, e.g. with TypeScript and generic types, sometimes the second example can be easier.

Thus I wouldn't have any rule around which one to use and which not anymore.